docs/TheoryDeepPixels.rst

```diff
@@ -0,0 +1,491 @@
+======================
+Theory of Deep Samples
+======================
+
+
+This document derives the techniques for splitting and combining two non-solid samples of
+equal depth and thickness. These should should be used by deep image "flattening" algorithms to compute the combined colour of two samples. The formulas are defined in the document "Interpreting OpenEXR Deep Pixels". This document derives those formulas, and is for information only.
+
+
+
+Definitions
+===========
+
+-  ratio of length of original sample to a subsample:
+   :math: 0\le r \le 1
```

Comment:
Looks like this isn't rendered properly, the ":math" appears in the output.

Reply:
Good catch! The whole list hadn't converted from tex properly (it was missing the symbols that were being defined), so I switched everything to a table.